misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobConsumer.kt

```diff
@@ -38,8 +38,12 @@ internal class SqsJobConsumer @Inject internal constructor(
 ) : JobConsumer {
   private val subscriptions = ConcurrentHashMap<QueueName, QueueReceiver>()
 
-  override fun subscribe(queueName: QueueName, handler: JobHandler) {
-    val receiver = QueueReceiver(queueName, handler)
+  override fun subscribe(queueNames: List<QueueName>, handler: JobHandler) {
```

Comment:
this doesn't need to take a list

Reply:
what are you proposing instead? we'd need to use a builder pattern or similar if we didn't take a list. or am i misunderstanding you?

Reply:
iterate outside. doesn't really matter

Reply:
we already do that. it has a different semantic though. calling subscribe multiple times with different queues (and the same handler) subscribes the same handler to different queues, with no prioritisation.